src/routes/components/Badge.tsx

```diff
@@ -117,6 +123,18 @@ export default function BadgePage() {
       description: "Includes an icon in the badge.",
       defaultValue: "false",
     },
+    {
+      name: "icontype",
```

Comment:
I could be wrong, but I don't think `icontype` was actually an option in LTS, pretty sure it's a new feature introduced with the latest versions. So these should be removed.